src/shared/auth/auth.context.ts

```diff
@@ -68,4 +68,9 @@ export class UmbAuthContext implements IUmbAuth {
 	signOut(): Promise<void> {
 		return this.#authFlow.signOut();
 	}
+
+	async isUserCurrentUser(userId: string): Promise<boolean> {
+		const currentUser = await firstValueFrom(this.currentUser);
```

Comment:
Is this different than `this.#currentUser.getValue()`    I see its async, but wouldn't this just give you undefined. Or would it wait for the first real value?

Reply:
It waits for the first real value. A small shortcut so you don't have to set up an observer etc.